source/gloperate/include/gloperate/pipeline/Slot.inl

```diff
@@ -68,6 +70,8 @@ Slot<T>::~Slot()
 template <typename T>
 bool Slot<T>::connect(Slot<T> * source)
 {
+    cppassist::debug(2) << this->qualifiedName() + ": connect slot " + source->qualifiedName();
```

Comment:
You should use repeated stream operators instead of string concatenation:
```cpp
cppassist::debug(2) << this->qualifiedName() << ": connect slot " << source->qualifiedName();
```

This applies for further loggings, too.